spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -119,10 +119,15 @@ def copy_value(self, orig_name, new_name):
 
     def load_data(self, filename, ext):
         try:
+            # TODO: Add dialog informing of possible override of variables
+            #       Check if user wants an override or wants to keep possible
+            #       all values
+            override = False
```

Comment:
This code should go outside the exception. Code within exceptions should be in general as little as possible to be granular in  what we want to catch. Also in this case `override = False`is something that will never fail.